internal/database/mariadb/init/schema.sql

```diff
@@ -169,6 +169,7 @@ create table if not exists ComponentInstance
     componentinstance_project              varchar(1024)                         null,
     componentinstance_pod                  varchar(1024)                         null,
     componentinstance_container            varchar(1024)                         null,
+    componentinstance_type                 enum('Unknown', 'Project', 'Server', 'SecurityGroup', 'DnsZone', 'FloatingIp', 'RbacPolicy', 'User')       not null,
```

Comment:
- I'd prefer to make the column nullable and set `Unknown` as default
- For the k8s assets, we also need a `Container` type.
- We wanted to model Openstack Projects with the Service entity. Is this `project` field referring to something else?